designs/octoplushy/src/head.mjs

```diff
@@ -613,15 +613,16 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
+    // eslint-disable-next-line no-unused-vars
     toAbs: (val, { options }, mergedOptions) =>
       ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
```

Comment:
Unrelated to the fix, but could use `Math.PI` here for clarity

Reply:
Done